libparsers/include/parsers/xml/stations.hpp

```diff
@@ -10,49 +10,53 @@
 
 namespace parsers {
 namespace xml {
-	namespace stations_and_stops {
+	namespace stations {
 
-		struct sub_message {
+		struct message {
 			std::string early;
 			std::string ontime;
 			std::string late;
 		};
+
 		struct probabilities {
 			std::uint8_t early = 0;
 			std::uint8_t ontime = 0;
 			std::uint8_t late = 0;
 		};
+
 		struct request_stop_marker {
 			enum class behaviour : std::uint8_t { fullspeed, normalbrake };
 			openbve2::datatypes::time early_time = 0;
+			bool using_early = false;
 			openbve2::datatypes::time late_time = 0;
+			bool using_late = false;
 			float distance = 0;
-			sub_message stop_message;
-			sub_message pass_message;
+			message stop_message;
+			message pass_message;
 			probabilities probability;
-			std::intmax_t max_cars = 0;
+			std::uint8_t max_cars = 0;
```

Comment:
I thought we discussed that max_cars should be a uintmax_t. Either way, it should be.